app/services/spotify_service.rb

```diff
@@ -1,6 +1,39 @@
+# frozen_string_literal: true
+
 class SpotifyService
-  def initialize(token)
-    @token = token
+  def initialize(user)
+    @user = user
+  end
+
+  def populate_playlist(playlist_id)
+    artists = Artist.select('artists.spotify_id, COUNT(artists.user_id) AS frequency')
```

Comment:
We should pull these out to the model as soon as we can, but looks good for now.